index.js

```diff
@@ -73,7 +74,7 @@ class WebTorrent extends EventEmitter {
     this.lsd = opts.lsd !== false
     this.torrents = []
     this.maxConns = Number(opts.maxConns) || 55
-    this.utp = opts.utp === true
+    this.utp = utp.UTP_SUPPORT ? opts.utp === true : false
```

Comment:
Rather than use a ternary operator, this is more clearly expressed as `utp.UTP_SUPPORT && opts.utp === true`

Reply:
👍